tests/tasks/setup.yml

```diff
@@ -0,0 +1,11 @@
+---
+- name: Make sure openssh is installed before creating backup
```

Comment:
Yes. One clean system (for example CI containers), we need to make sure the openssh is installed before we backup the package-provided files, otherwise we would "backup" missing file, which would be restored after first test. Indeed, the role itself has its own "check" to install required packages.